src/PKSim.Infrastructure/ProjectConverter/v5_2/CompoundConverter.cs

```diff
@@ -69,7 +69,6 @@ private void addNewParameters(Compound compound)
          compound.Add(_cloner.Clone(defaultCompound.Parameter(ConverterConstants.Parameter.BP_AGP)));
          compound.Add(_cloner.Clone(defaultCompound.Parameter(ConverterConstants.Parameter.BP_ALBUMIN)));
          compound.Add(_cloner.Clone(defaultCompound.Parameter(ConverterConstants.Parameter.BP_UNKNOWN)));
-         compound.Add(_cloner.Clone(defaultCompound.Parameter(ConverterConstants.Parameter.CalculatedSpecificIntestinalPermeability)));
```

Comment:
This parameter does not exist in compound anymore. (this is a simulation parameter). remove